src/Shared/WebEncoders/WebEncoders.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+#if NETCOREAPP3_0
```

Comment:
FYi @BrennanConroy I had to add this guard around the using buffers introduced from the spanify web encoders change last week since it isn't available in netstandard 2.0

Reply:
Makes sense 👍